.github/workflows/matrix.yml

```diff
@@ -243,9 +243,28 @@ jobs:
         restore-keys: |
           ccache-${{ runner.os }}-${{ matrix.compiler }}--
     - uses: ammaraskar/gcc-problem-matcher@master
-    - name: build and test
+    - name: build
       if: ${{ env.SKIP == 'false' }}
-      run: bash ./build-scripts/build.sh
+      run: bash ./build-scripts/gha_compile_only.sh
+    - name: retrieve ccache debug files
+      run: |
+        find . -type f -name "*cache-*" -exec tar -rvf ccache.debug.tar {} \;
+    - name: upload ccache debug files
+      uses: actions/upload-artifact@v2
+      with:
+        name: ccache-debug
+        path: ccache.debug.tar
+        if-no-files-found: ignore
+    - name: cleanup ccache before upload
+      if: ${{ env.skip == 'false' && !failure() && (runner.os == 'Linux' || runner.os == 'macOS') }}
```

Comment:
```suggestion
      if: ${{ env.SKIP == 'false' && !failure() && (runner.os == 'Linux' || runner.os == 'macOS') }}
```

Reply:
You'll delete the original at the bottom if you move this up here?

Reply:
Yes I forget to delete the old one.